tool/e2e-tests/file-upload-info/interfaces/FileUploadInfo.interface.ts

```diff
@@ -0,0 +1,7 @@
+import {FileUploadType} from "../../enums/FileUploadType.enum"
```

Comment:
Could `FileUploadType.enum.ts`, this, and the two `-info.ts` files just exist in one directory for simplicity?